pkg/network/setup/netpod/discover.go

```diff
@@ -96,8 +96,14 @@ func (n NetPod) discover(currentStatus *nmstate.Status) error {
 				return err
 			}
 
-		// Skip the discovery for all other known network interface bindings.
 		case vmiSpecIface.Binding != nil:
+			if podIfaceExists {
```

Comment:
Please add a comment here that explains why it ignores cases where the pod interface is missing.
(this is the case with the other bindings, making this "special")